wkcuber/api/Layer.py

```diff
@@ -64,9 +59,15 @@
 
 
 class Layer(Generic[MagT]):
+    """
+    A `Layer` consists of multiple `wkcuber.api.MagDataset`s, which store the same data in different magnifications.
+    """
 
     COLOR_TYPE = "color"
     SEGMENTATION_TYPE = "segmentation"
+    """
+    There are two different types (`COLOR_TYPE`, `SEGMENTATION_TYPE`).
+    """
```

Comment:
Not sure if that's useful here. Also, they are only used together with `Dataset.add_layer`, so they should be defined there rather.

```suggestion
```